src/domain/homepage/Homepage.test.tsx

```diff
@@ -56,7 +58,7 @@ describe('Create new hanke from dialog', () => {
     await user.click(screen.getByRole('button', { name: /luo hanke/i }));
 
     expect(screen.getByText(/kentän pituus oltava vähintään 3 merkkiä/i)).toBeInTheDocument();
-    expect(screen.getAllByText(/kenttä on pakollinen/i)).toHaveLength(2);
+    expect(screen.getAllByText(/kenttä on pakollinen/i)).toHaveLength(1);
```

Comment:
Should the email field be cleared to test the validation on it?

Reply:
Good point. Cleared email field.